specification/netapp/resource-manager/readme.md

```diff
@@ -111,7 +120,6 @@ swagger-to-sdk:
   - repo: azure-sdk-for-js
   - repo: azure-sdk-for-go
   - repo: azure-sdk-for-java
-  - repo: azure-sdk-for-trenton
```

Comment:
Can we know why remove this entry?

Reply:
Hi, I missed that entry, I am not sure what it is can I ask what is it for?
Let me know if it is needed and I will add it again.

Reply:
Do you need TerraForm? it yes. you need add it back.

Reply:
Thank you, I added it back.